lms/djangoapps/grades/rest_api/v1/tests/test_gradebook_views.py

```diff
@@ -1210,6 +1211,17 @@ def test_filter_course_grade_absent(self, login_method, num_enrollments):
                 self.assertEqual(actual_data['total_users_count'], num_enrollments)
                 self.assertEqual(actual_data['filtered_users_count'], num_enrollments)
 
+                mock_grade.side_effect = mocks
```

Comment:
This test is already a bit monstrous; could you split out some submethods for the different sections of it?